### PR TITLE
Comment book

### PR DESCRIPTION
**Pull request by pdkproitf**

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Book < ActiveRecord::Base
   belongs_to :category
   belongs_to :publish
-  has_many :comments, dependent: :destroy
+  has_many :comments
```
Comment (yoyocoder) [inline app/models/book.rb]: If you delete the book, do you need `comments` exist in database? If not, please re-add `dependent: :destroy`.

File: app/models/category.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Category < ActiveRecord::Base
-  has_many :books, dependent: :destroy
+  has_many :books
   validates :name,  presence: true,
                     length: { maximum: 50,
                               message: "can't be longer than 50 characters" },
```
Comment (yoyocoder) [inline app/models/category.rb]: Please reformat your code. I feel `nhức nhói`.

File: app/models/book.rb (hunk 1 above)
Comment (pdkproitf) [inline app/models/book.rb]: got it. i thought that. may be i forgot it.

**Comment by yoyocoder** [comment]
👍

**Pull request merged**